recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "41.0.1" %}
+{% set version = "41.0.2" %}
 
 package:
   name: cryptography
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/cryptography/cryptography-{{ version }}.tar.gz
-  sha256: d34579085401d3f49762d2f7d6634d6b6c2ae1242202e860f4d26b046e3a1006
+  sha256: 7d230bf856164de164ecb615ccc14c7fc6de6906ddd5b491f3af90d3514c925c
 
 build:
   number: 0
```

Comment:
I think this would be the proper way to handle ppc64le failures in 1.1.1:
```
build:
  skip: true. # [ppc64le and openssl == 1.1.1]
requirements:
  host:
    - openssl {{openssl}}

Reply:
I tested this and got the following:
```
Error: Invalid selector in meta.yaml line 17:
offending line:
  skip: true  # [arm64 and openssl == 1.1.1]
exception:
unexpected EOF while parsing (<string>, line 1)
```
I don't think the preprocessor has access to variant information.  I will do this as a jinja template instead.